.github/workflows/ci.yml

```diff
@@ -4,36 +4,38 @@ on: [push, pull_request]
 
 jobs:
 
-  docs:
+  # The docs action is currently replaced by a build with CircleCI (see .circleci/config.yml)
+  # When/if GitHub actions allows for storage of (unzipped) artifacts, we can move back to this action
+  # docs:
 
-    runs-on: ubuntu-latest
-    timeout-minutes: 30
+  #   runs-on: ubuntu-latest
+  #   timeout-minutes: 30
 
-    steps:
-    - uses: actions/checkout@v2
+  #   steps:
+  #   - uses: actions/checkout@v2
 
-    - name: Set up Python 3.7
-      uses: actions/setup-python@v1
-      with:
-        python-version: 3.7
+  #   - name: Set up Python 3.7
+  #     uses: actions/setup-python@v1
+  #     with:
+  #       python-version: 3.7
 
-    - name: Install system dependencies
-      # remove occasionally problematic repositories we don't use anyway
-      run: |
-        sudo rm -f /etc/apt/sources.list.d/dotnetdev.list /etc/apt/sources.list.d/microsoft-prod.list
-        sudo apt update
-        sudo apt install texlive-base texlive-generic-recommended texlive-fonts-recommended texlive-latex-base texlive-latex-recommended texlive-latex-extra dvipng dvidvi
+  #   - name: Install system dependencies
+  #     # remove occasionally problematic repositories we don't use anyway
+  #     run: |
+  #       sudo rm -f /etc/apt/sources.list.d/dotnetdev.list /etc/apt/sources.list.d/microsoft-prod.list
+  #       sudo apt update
+  #       sudo apt install texlive-base texlive-generic-recommended texlive-fonts-recommended texlive-latex-base texlive-latex-recommended texlive-latex-extra dvipng dvidvi
```

Comment:
Note here that the packages for a PDF build are installed. But I'm pretty sure this action doesn't actually build latex, only html? So I haven't added them to the circleci config

Reply:
Or unless there is another reason they are required?

Reply:
I think this is left-over and probably not needed, but I suggest that we just ignore that as part of this change.

Reply:
removed in 832600a